src/roiextractors/extractors/hdf5imagingextractor/hdf5imagingextractor.py

```diff
@@ -71,13 +74,16 @@ def __init__(
         (
             self._num_channels,
             self._num_frames,
-            self._size_x,
-            self._size_y,
+            self._rows,
+            self._columns,
         ) = get_video_shape(self._video)
```

Comment:
`get_video_shape` doesn't really do much TBH

How about just doing the aliasing/assignment as

```python
self._num_channels, self._num_frames, self._num_rows, self._num_columns = self._video.shape
```

Reply:
Btw, do you know where this assumption comes from? that videos in this format come in that shape? is there a specification?

Reply:
So I implemented this.